plugins/reporters/web-app-template/src/components/VulnerabilitiesTable.jsx

```diff
@@ -168,28 +168,93 @@ const VulnerabilitiesTable = ({ webAppVulnerabilities = [], showExcludesColumn =
                 text: (<VulnerabilityRatingTag severity={'low'}/>),
                 value: 3
             },
-            {
-                text: (<VulnerabilityRatingTag severity={'resolved'}/>),
-                value: 5
-            },
             {
                 text: (<VulnerabilityRatingTag severity={'none'}/>),
                 value: 4
+            },
+            {
+                text: (<Tag color="#b0c4de">Resolved</Tag>),
+                value: 10
             }
         ],
         filteredValue: filteredInfo.severityIndex || null,
-        onFilter: (value, record) => record.severityIndex === Number(value),
+        onFilter: (value, record) => record.severityIndex === Number(value)
+            || (record.severityIndex > 10 && Number(value) >= 10),
         render: (text, record) => (
             record.isResolved
                 ? (
-                    <Tooltip
-                        placement="right"
-                        title={Array.from(record.webAppVulnerability.resolutionReasons).join(', ')}
-                    >
-                        <IssuesCloseOutlined
-                            className="ort-ok"
-                        />
-                    </Tooltip>
+                    <span>
+                        {
+                            record.severityIndex === 10
+                            && (
+                                <VulnerabilityRatingTag
+                                    isResolved={true}
+                                    severity={'critical'}
+                                    tooltipText={`Resolved with ${
+                                        Array.from(record.webAppVulnerability.resolutionReasons).join(', ')
+                                    } resolution${
+                                        record.webAppVulnerability.resolutionReasons.size > 0 ? 's' : ''
+                                    }`}
```

Comment:
Maybe the generation of the "Resolved with" string could be extracted to a function that just takes the array of reasons, for reuse below?